doc/rados/command/list-inconsistent-obj.json

```diff
@@ -0,0 +1,140 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "epoch": {
+      "description": "Scrub epoch",
+      "type": "integer"
+    },
+    "inconsistents": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "object": {
+            "description": "Identify a Ceph object",
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string"
+              },
+              "nspace": {
+                "type": "string"
+              },
+              "locator": {
+                "type": "string"
+              },
+              "snap": {
+                "oneOf": [
+                  {
+                    "type": "string",
+                    "enum": [ "head", "snapdir" ]
+                  },
+                  {
+                    "type": "integer",
+                    "minimum": 0
+                  }
+                ]
+              }
+            },
+            "required": [
+              "name",
+              "nspace",
+              "locator",
+              "snap"
+            ]
+          },
+          "errors": {
+            "description": "Union of all shard errors, including errors in which the faulty shard is NOT identified",
+            "type": "array",
+            "items": {
+              "enum": [
+                "attr_unexpected",
+                "missing",
+                "stat_error",
+                "read_error",
+                "data_digest_mismatch",
+                "omap_digest_mismatch",
```

Comment:
I think here I'd expect that data_digest_mismatch would be present whenever the shards' data digests do not match (ditto for omap).  For an EC pool, this is a bit more complicated because there are two possible mismatches:
1) for shard <x>, shard <x>'s locally stored hash does match the computed hash (not the object_info hash)
2) shard <x>'s stored hash does not match shard <y>'s stored hash (note, each shard stores all of the hashes for all of the shards)
I think 2 should cause data_digest_mismatch and 1 should have another error type (data_digest_mismatch_ec? or a better name...)